crossing-repair.lic

```diff
@@ -178,7 +178,7 @@ class CrossingRepair
   def setup
     arg_definitions = [
       [
-        { name: 'town', regex: /Crossing|River.*|Theren.*|Shard|Hib.*|Ratha/i, optional: true, description: 'Town to sell in' },
+        { name: 'town', regex: /^(?!force|debug).+$/, optional: true, description: 'Town to sell in' },
```

Comment:
What do you think about using `DRC.$HOMETOWN_REGEX` (or maybe you can just use `$HOMETOWN_REGEX`?) here?

Reply:
I'll try that out, thanks

Reply:
```suggestion
        { name: 'town', regex: $HOMETOWN_REGEX, optional: true, description: 'Town to sell in' },
```

Reply:
This change depends on https://github.com/rpherbig/dr-scripts/pull/5345